isolate_example/lib/page_three.dart

```diff
@@ -235,7 +255,12 @@ Widget createButtons(BuildContext context) {
           RaisedButton(
             child: const Text('Transfer Data to 2nd Isolate'),
             elevation: 8.0,
-            onPressed: controller.generateRandomNumbers,
+            onPressed: () => controller.generateRandomNumbers(false),
+          ),
+          RaisedButton(
```

Comment:
It might be nice to provide some feedback to the user when they tap a button. 
When I ran this, I wasn't sure which one I had pressed - this could be informed by highlighting that button, or displaying text that indicates what is being executed.
It also seemed that the other buttons could not be pressed while a process is running. It might be a good idea to disable them if that is the case.

Reply:
@bratay It's always good to either respond to a comment or mark it resolved when you make the appropriate code change. 😄